test/reactionsystem_core/reactionsystem_structure.jl

```diff
@@ -747,14 +747,9 @@ end
 
 # Tests construction of empty reaction networks.
 let
-    function unpacksys(sys)
-        get_eqs(sys), get_iv(sys), get_unknowns(sys), get_ps(sys), nameof(sys), get_systems(sys)
-    end
-
-    empty_network_1 = @reaction_network
-    eqs, iv, ps, name, systems = unpacksys(empty_network_1)
-    @test length(eqs) == 0
-    @test nameof(iv) == :t
-    @test length(get_unknowns(empty_network_1)) == 0
-    @test length(ps) == 0
+    empty_network = @reaction_network
+    @test length(equations(empty_network)) == 0
+    @test nameof(independent_variable(empty_network)) == :t
```

Comment:
This is a deprecated function. More generally why have you dropped using the getters here? We need to test them too.

Reply:
https://github.com/SciML/ModelingToolkit.jl/blob/e16c18091ac71ad45b2e87d4028e015deb70f713/src/systems/abstractsystem.jl#L319

Reply:
Didin't realise that the test was specific to using the `get_eqs` getter instead of the `equations` getters. Figured it was just about checking the content of the empty network. If this is and important part of the test, shouldn't we just use the full set? 

Good point about `independent_variable`, I will revert that and the others.